common/lib/xmodule/xmodule/js/src/word_cloud/word_cloud_main.js

```diff
@@ -250,10 +268,17 @@ define('WordCloudMain', [], function () {
                 .attr('transform', 'translate(' + (0.5 * this.width) + ',' + (0.5 * this.height) + ')')
                 .selectAll('text')
                 .data(words)
-                .enter().append('g');
+                .enter()
+                .append('g')
+                .attr('aria-labelledby', function(d) {
```

Comment:
This is invalid. Also make sure that IDs are unique (even with multiple word clouds on the same page, possibly with the same words).

![image](https://cloud.githubusercontent.com/assets/484484/17380637/1da787b8-5996-11e6-9363-f730fb6f447c.png)